### ISSUE TITLE
batch processing 

### ISSUE THREAD

**Issue by SemElVik** [tezos-reward-distributor-organization/tezos-reward-distributor#230]
**Describe the bug**
I have 407 transactions for payment.
batch 1 - ok  (Block 983,061)    
batch 2 - ok  (Block 983,063) 
batch 3 - does not include in the chain. 

this hash (batch 3) is still pending inclusion in the blockchain. 
it just hangs in anticipation of the inclusion of the transaction hash and does not generate a report on the completion of the payment.
(sufficient balance to pay)

[1.txt](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/files/4733945/1.txt)

payment was to be made in block 983,065., but based on experience with TRD, it most likely will not be.

how do you propose to handle this batch now? 

**Environment**
 - Source Code Build (TRD version 5.13)
 - Chain mainnet
 - python3 src/main.py -C 234 -M 3

**Comment by SemElVik** [comment]
Crtl+C 

```
2020-06-05 08:50:52,924 - consumer0 - Operation ooj5fLJwjhfL9zKyc7NyLe2CaVLBkrtfLNEF25tcZ7GkUpoHrBt wait is timed out. Not sure about the result!
2020-06-05 08:50:52,925 - consumer0 - Payment of batch 3 is complete, in 1 attempt(s)
2020-06-05 08:50:52,925 - consumer0 - Processing completed for 407 payment items.
    elif lock.acquire(block, timeout):
KeyboardInterrupt
```
 *paid manually*

**Comment by dansan566** [comment]
We once had a similar issue: https://gitlab.com/tezos/tezos/-/issues/656
Because you don't pay the reactivation fee, I would rule that out. 
What does "paid manually" mean? Did you use retry_failed.py? Was there a file created in payments/failed? And there was no file in payments/done? You also waited for 20 blocks after the cycle ended. Very strange. This needs some exploring. Did you create a log?

**Comment by SemElVik** [comment]
1. since I have only 7 transactions in batch3, I calculated their addresses and amounts myself. paid by package through kukai; 
therefore, I launched 7 transactions in manual mode through a regular wallet (package).

2. The file was not created in the `payments/failed` folder.
the file was created, `.../payments/done/234.csv`; the file was created after I stopped the _Ctr+С_ , and also, although the file was created, it turned out to be **empty**.
3. after the end of the cycle, I waited for batch3 ~100 blocks. (before stopping).

4. The log is written by default, and I have it.
I'm not sure if I can post it here (if need it, I can send it personally). 
after batch1+batch2,,  batch3 was not broadcast to the blockchain on my node. just missed it.
I have changed my transaction (increased) fee and will keep track of the next payout.

all this is strange. how batch can get lost and not broadcast....
